lib/px/utils/pxtimer.lua

```diff
@@ -29,5 +31,10 @@ function M.application(file_name)
 	    return
 	end
 	submit_on_timer()
+
+	function M.get_time_in_milliseconds()
+		px_logger.error('current time in milliseconds ' .. socket.gettime() * 10000)
```

Comment:
Two comments here: 

1. You multiply by 10000 (not 1000) in the first line.
2. Instead of calling `socket.gettime()` twice, just save `socket.gettime() * 1000` to a local variable.